examples-json/questions.json

```diff
@@ -1,6 +1,24 @@
 [
   {
-    "id": "managed-question-asset-inventory-count",
+    "title": "New Question",
+    "queries": [
+      {
+        "query": "Find Person with lastName='Jones'"
+      },
+      {
+        "query": "Find compliance records"
```

Comment:
This isn't actually a valid query, `Find Record` or `Find Document` would work, see https://docs.jupiterone.io/en/latest/index.html for the data model with the class names.

Reply:
Gotcha. This was just a test case that I made up